### ISSUE TITLE
framebuffer not visible

### ISSUE THREAD

**Issue by gapid-user** [google/gapid#983]
first error from log:

gapis: Framebuffer error: No format set (3 times)

Then a bunch of command errors

**Comment by gapid-user** [comment]
I also noticed that there are two contexts from two different threads. One thread called "Renderthread", I don't know what that is. Maybe that's related to the problem.

**Comment by ben-clayton** [comment]
Hi @gapid-user (nice name),
Would you be happy sharing the `.gfxtrace` file with me?

**Comment by gapid-user** [comment]
sure, here it is:
[poolgame_v11_20170824_1436.zip](https://github.com/google/gapid/files/1248816/poolgame_v11_20170824_1436.zip)

**Comment by ben-clayton** [comment]
Thank you @gapid-user,

So it looks like we never saw the call to `eglCreateContext`. We've seen this happen on an oldish Samsung device before, and so we added some code that tries to recover from missing context. Unfortunately this path had a bug that made things explode. Just to confirm - is this a galaxy tab 2?

There's another issue with this trace where pre-compiled shaders were not disabled. We've recently fixed an issue where this wasn't disabled by default.

I'm planning on pushing out a release in the next few working days. Both of these issues should be fixed for that version.

Thanks again,
Ben
